packages/pyright-internal/src/tests/samples/reportAny.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Callable
```

Comment:
What about 
```py
a: list[Any]
print(a)  # what do we see?
```

Reply:
no error, but i think it's fine because you will still get an error if you try to use any of the `Any`s in the list

Reply:
actually it didnt work at all but its fixed now

Reply:
actually it still sucks because there's too much stuff in the stdlib that contains `Any` (it even complains about the `cast` function), so im just gonna raise a separate issue for this

Reply:
#82